parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -97,6 +97,11 @@ if [[ "${GPU_NODE}" = true ]]; then
         installGPUDrivers
     fi
     ensureGPUDrivers
+    if [[ "${ENABLE_GPU_DEVICE_PLUGIN_IF_NEEDED}" = true ]]; then
+        systemctlEnableAndStart nvidia-device-plugin || exit $ERR_GPU_DEVICE_PLUGIN_START_FAIL
```

Comment:
add some logging? or we dont need since we have the verbose tuned on?

Reply:
added some logging for failures